portfoliostyle.css

```diff
@@ -0,0 +1,190 @@
+html {
+	font-size: 10px;
+	box-sizing: border-box;
+	scroll-behavior: smooth;
+}
+body {
+	margin: 0;
+
+}
+
+
+h1 {
+	font-size: 7rem;
+	color:  #FFFF00;
+	font-family: 'Anton', sans-serif;
+
+}
+
+h2 {
+	font-size: 4rem;
+	color: #b1fae2;
+	font-family: 'Anton', sans-serif;
+
+}
+
+h3 {
+	font-size: 2.9rem;
+	color: #FFFF00;
+	padding-bottom: 10px;
+	font-family: 'Anton', sans-serif;
+}
+
+p {
+	font-size: 2rem;
+	font-family: 'Exo 2', sans-serif;
+	color: #f7faf9;
+}
```

Comment:
for the font-family, you should write it at once for all of these so that if you change your mind you only need to change it in one place:

```css
h1, h2, h3 {
   font-family: 'Anton';
}